index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>API Project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+
+</head>
+
+<body>
+    <div class="container">	
+
+        <div class="user-container">
+        </div>
+
+        <div class="quote-container">    
+        </div>
+
+        <div class="content-container">	
+            <div class="pokemon-container"></div>	
+            <div class="meat-container"></div>	
+        </div>
+
+        <div class="friends">	
+            <h4>Friends</h4>	
+            <div class="friends-container">	
+            </div>	
+        </div>	
+
+        <div class="buttons">	
+            <button id="data">Load User Data</button>	
+            <button id="view">Display User</button>	
+        </div>	
+
+    </div>
+    
+    <script id="details-template" type="text/x-handlebars-template">
+        <div>
+            <img id="profile-pic" src={{picture}}>
+        </div>
+        <p class="user-info">{{name}}</p>
+        <p class="user-info">{{address}}</p>
+    </script>
+
+    <script id="friends-template" type="text/x-handlebars-template">
```

Comment:
{{#each friends}} will loop over what you send to the template. Look at the documents to see all the capabilities of whatever you are using